tests/components/homematicip_cloud/test_init.py

```diff
@@ -32,34 +30,11 @@
     assert len(mock_config_entries.flow.mock_calls) >= 2
 
 
-async def test_config_already_registered_not_passed_to_config_entry(hass):
```

Comment:
I think we still need a test where the config yaml tries to set up the same entry already added, and the existing entry doesn't have a unique_id.

Create a mock entry without unique_id, add it to hass, then pass the corresponding config yaml to `async_setup_component`. Then assert that the same mock instance is the only entry instance for the integration.

Reply:
i re-added the test and rewrote it.